features/importing/archivist.feature

```diff
@@ -10,37 +9,38 @@ Feature: Archivist bulk imports
   
   Scenario: Make a user an archivist
   
-  Given I have pre-archivist setup for "elynross"
+  Given I have pre-archivist setup for "archivist"
     And the default ratings exist
   When I am logged in as an admin
-    And I make "elynross" an archivist
+    And I make "archivist" an archivist
   Then I should see "User was successfully updated"
     
   Scenario: Archivist can see link to import for others
   
-  Given I have an archivist "elynross"
+  Given I have an archivist "archivist"
     And the default ratings exist
-  When I am logged in as "elynross"
+  When I am logged in as "archivist"
     And I go to the import page
   Then I should see "Import for others ONLY with permission"
 
-  Scenario: Import a single work as an archivist and send the right email
+  Scenario: Importing for an author without an account should have the correct byline and email
   
-  Given I have an archivist "alice_ttlg"
+  Given I have an archivist "archivist"
     And the default ratings exist
-  When I am logged in as "alice_ttlg"
+  When I am logged in as "archivist"
     And I import the work "http://rebecca2525.livejournal.com/3562.html"
   Then I should see "We have notified the author(s) you imported works for"
     And I should see "Importing Test"
+    And I should see "rebecca2525 [archived by archivist]"
```

Comment:
Since the naming changes might be obscuring this, here is where it's testing for the presence of the archivist byline on a work whose author doesn't have an AO3 account